### PR TITLE
add optional parameter 'aws_token' to unload function

### PR DESCRIPTION
**Pull request by rynmccrmck**
## Description & motivation
Allow compatibility with session token for use with temporary access credentials.

## Checklist
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)
- [ ] I have added tests & descriptions to my models (and macros if applicable)

### CODE REVIEW

**Comment by yyf6413** [comment]
👍

**Comment by rynmccrmck** [comment]
@clrcrl if there's anything else I can do or provide please let me know

File: macros/unload.sql [hunk 1]
```diff
@@ -56,6 +57,9 @@ where option is
   {% elif aws_key and aws_secret %}
   ACCESS_KEY_ID '{{ aws_key }}'
   SECRET_ACCESS_KEY '{{ aws_secret }}'
+  {% if aws_token %}
+    SESSION_TOKEN '{{ aws_token }}'
+  {% endif %}
```
Comment (jtcohen6) [inline macros/unload.sql:62]: Just checked [the docs](https://docs.aws.amazon.com/redshift/latest/dg/copy-parameters-authorization.html):

> When SESSION_TOKEN is specified, you must also use ACCESS_KEY_ID and SECRET_ACCESS_KEY to provide temporary access key credentials

Makes sense!

**Review by jtcohen6 — APPROVED** [review]
Sorry for the (long) delay @rynmccrmck, this looks great! Thanks for the contribution :)

**Pull request merged**